tests/test_fixtures.py

```diff
@@ -41,6 +42,14 @@ def test_env_old_fixture(short_env, short_test):
         spawn.close()
         assert spawn.exitstatus == 0
 
+def test_env_env_fixture(short_env, short_test):
+    env=os.environ.copy()
```

Comment:
Hmm, I wonder if this is necessary. Can you just set env["LG_ENV"] without importing the outer environment? I could imagine that importing it could leak some unneeded variables that could influence the test.

Reply:
When no env argument is given, pexpect.spawn() executes with os.environ, so I am not introducing any leakage that wasn't there before. Also, I really do need at least some of the existing os.environ, as executing without fx. PATH really does not work as expected.